deps/3rdparty/libdispatch/src/queue.c

```diff
@@ -665,6 +679,27 @@ dispatch_queue_create(const char *label, dispatch_queue_attr_t attr)
 #endif
 }
 
+void _dispatch_queue_specific_release(dispatch_queue_t dq) {
+	dispatch_queue_specific_list_t dqsl = dq->dq_specific_q;
+	dispatch_queue_specific_t lastvar = NULL;
+	dispatch_queue_specific_t var;
+
+	TAILQ_FOREACH(var, &dqsl->contextList, specific) {
+		if (lastvar) {
```

Comment:
You need to do this because var contains `var->next`, right?
There may be another TAILQ macro that handles this, but perhaps not.